test/Tools/NodeDocumentationMarkdownGeneratorTests/MarkdownGeneratorCommandTests.cs

```diff
@@ -146,6 +146,7 @@ public void ProducesCorrectOutputFromCoreDirectory_preloadedbinaries()
          
```

Comment:
Maybe you dont have to include "Autodesk.DesignScript.Geometry.DebugTools.DisableAsmJournaling.md", "Autodesk.DesignScript.Geometry.DebugTools.EnableAsmJournaling.md",  as part of the list?

Reply:
When testing in my local environment I got 756 md files including those two one that's why I included them, not sure about which nodes they refer.
My only guess about why that test is failing could be related to the decoration added in src/Libraries/DSOffice/Excel.cs file, probably is if they are decorated with `[NodeObsolete("ExcelInteropObsoleteNode", typeof(Properties.Resources))] `the md file is not generated but not sure.